tests/js/acs.scrypttest.js

```diff
@@ -0,0 +1,72 @@
+const { expect } = require('chai');
+const {
+  bsv,
+  buildContractClass,
+  getPreimage,
+  Ripemd160,
+  toHex,
+  SigHashPreimage,
+} = require('scryptlib');
+
+const {
+  compileContract,
+  inputIndex,
+  inputSatoshis,
+  newTx,
+} = require('../../helper');
+
+// Demo values for testing
+const pkhX = 'f5af74378a5faaff242f47490fd225868586f6b2';
+const addressX = 'n3v22oq3CpnCZnVYFcXP6GUaXt7Ts2TCTj';
```

Comment:
Let's dynamically generate these as u did in deployment.

Reply:
Have modified it. I have used the static values for the preimage and txContext in launch.json. But it's no longer necessary to keep the static values in the test script.